src/index.tsx

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
 import reportWebVitals from './reportWebVitals';
 
-ReactDOM.render(
+import App from "./App";
+import {createRoot} from "react-dom/client";
+ 
+const rootElement = document.getElementById("root");
+if (!rootElement) throw new Error('Failed to find the root element');
```

Comment:
Ts som gnällde på denna? Den kommer aldrig ske såvida ingen går in i html och tar bort root